eolearn/io/raster_io.py

```diff
@@ -406,18 +409,19 @@ class ImportFromTiffTask(BaseRasterIoTask):
     def __init__(
         self,
         feature: Feature,
-        folder: str,
+        path: str = ...,  # type: ignore[assignment]
         *,
         use_vsi: bool | None = None,
         timestamp_size: int | None = None,
         filesystem: FS | None = None,
         image_dtype: np.dtype | type | None = None,
         no_data_value: float | None = None,
         config: SHConfig | None = None,
+        folder: str | None = None,
     ):
         """
         :param feature: EOPatch feature into which data will be imported
-        :param folder: A directory containing image files or a path of an image file
+        :param path: A directory containing image files or a path of an image file
```

Comment:
The same comment about the `filesystem` parameter should be given here, no?

```suggestion
        :param path: Path pointing to an image file or to a directory of image files to be imported. If `filesystem` is not defined, absolute paths should be provided.
```